docs/include/conf/portland/visiting.rst

```diff
@@ -72,6 +78,11 @@ There are two bike rental companies in downtown Portland that may be good option
 - `Cycle Portland <http://www.portlandbicycletours.com/>`__
 - `Biketown <https://www.biketownpdx.com>`__
 
+Sundries
+--------
+- `Whole Foods <https://www.google.com/maps/place/Whole+Foods+Market/@45.5233154,-122.6858396,17z/data=!3m1!4b1!4m5!3m4!1s0x54950a0250842545:0xda581c4844197daf!8m2!3d45.5233154!4d-122.6836509?hl=en-us>`__ and `Target <https://www.google.com/maps/place/Target/@45.5201104,-122.6840493,17z/data=!3m1!4b1!4m5!3m4!1s0x54950a0497e34c3b:0x3bbb7b17d08ba5b0!8m2!3d45.5201104!4d-122.6818606?hl=en-us>`__ are less than 10 minutes' walk from the Crystal Ballroom. Target also has a CVS Pharmacy inside.
+- `Apple Pioneer Place <https://www.google.com/maps?hl=en-us&client=safari&yv=3&lqi=CgthcHBsZSBzdG9yZSIDiAEB&um=1&ie=UTF-8&fb=1&gl=us&entry=s&sa=X&ftid=0x54950a056f058c1f:0xe391f5206cbeea58&gmm=CgIgAQ%3D%3D>`__ and `Microsoft Store <https://www.google.com/maps/place/Microsoft/@45.51751,-122.678518,17z/data=!3m1!4b1!4m5!3m4!1s0x54950a0578ff9e25:0x99b82e842d6d8f4f!8m2!3d45.51751!4d-122.6763293?hl=en-us>`__ are both conveniently located in downtown Portland.
```

Comment:
Are these really sundries?

Reply:
I odn't feel strongly on this either way. People at conferences buying laptop chargers is not entirely uncommon...